client/my-sites/media-library/content.jsx

```diff
@@ -247,6 +295,10 @@ const MediaLibraryContent = React.createClass( {
 
 export default connect( ( state, ownProps ) => {
 	return {
-		siteSlug: ownProps.site ? getSiteSlug( state, ownProps.site.ID ) : ''
+		siteSlug: ownProps.site ? getSiteSlug( state, ownProps.site.ID ) : '',
+		connectedServices: toArray( getKeyringConnections( state ) ).filter( item => item.type === 'other' && item.status === 'ok' ),
```

Comment:
My understanding is that both publicize and non-publicize services use the same underlying keyring mechanism (I've just connected twitter and gphotos and both are stored in the keyring). If our goal is to determine whether the connections were already requested to avoid doing it again, shouldn't we take into account the publicize ones as well? Or maybe our goal is to determine whether `gphotos` is already there; in that case, would be better to check for `service=google_photos` instead of `type=other`?

I'm trying to understand whether we may run into some race condition if, in the future, more keyring services (`other` type) are added but they are unrelated to the media library, so the developers don't need to check this code, hence they may break this inadvertently.